src/bp/nlu-core/predict-pipeline.ts

```diff
@@ -76,23 +76,19 @@ async function DetectLanguage(
   const supportedLanguages = Object.keys(predictorsByLang)
 
   const langIdentifier = LanguageIdentifierProvider.getLanguageIdentifier(tools.mlToolkit)
-  const lidRes = await langIdentifier.identify(input.sentence)
-  const elected = lidRes.filter(pred => supportedLanguages.includes(pred.label))[0]
-  let score = elected?.value ?? 0
+  const possibleMlLangs = await langIdentifier.identify(input.sentence)
+  const bestMlLangMatch = possibleMlLangs[0]
```

Comment:
I don't see the point of declaring a variable for possibleMLLangs  if you don't use it later 

```
const bestMatch = (await langIdentifier.identify(input.sentence))[0]
```

Reply:
Changed for `(await  ...)[0]`